tests/System.Text.Http.Tests/GivenAnHttpHeaders.cs

```diff
@@ -54,11 +54,13 @@ public void Its_enumerator_Current_returns_the_same_item_until_MoveNext_gets_cal
             enumerator.MoveNext();
 
             var current = enumerator.Current;
-            current.Should().Be(enumerator.Current);
+            //TODO: no spans on the heap
```

Comment:
I can take care of this by comparing the pair elements individually. This TODO will go away.
I was not sure if that is interesting considering that the test is disabled.